tests/testFileEditor.js

```diff
@@ -212,6 +215,20 @@ const verifyEditData = [
 describe('test file editor', function() {
     this.timeout(20000);
 
+    describe('test base64-util lib, on which file editor depends', function() {
```

Comment:
I'd prefer a test of the editor itself with such a string so that we don't rely on current implementation details.

Reply:
done